src/app/public/modules/text-editor/services/text-editor-adapter.service.ts

```diff
@@ -538,4 +525,13 @@ export class SkyTextEditorAdapterService {
       .replace(/&amp;/g, '&');
   }
 
+  private toggleEditorAbility(id: string, focusableChildren: HTMLElement[], textEditorNativeElement: any, disabled: boolean): void {
+    textEditorNativeElement.style.pointerEvents = disabled ? 'none' : 'auto';
+    textEditorNativeElement.setAttribute('aria-disabled', disabled.toString);
```

Comment:
`disabled.toString` needs a `()` at the end or it will just print out `function toString() { ... }`

Reply:
Not sure what this will do if `disabled` is `null`, but it might be safer just to resolve this to a string:
`disabled ? 'true' : 'false'`

Reply:
ahhh good catches. will fix!

Reply:
fixed in commit: ["use elvis operator"](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/9566544a6d68e92e7e6b570f12699a2ce7b13164)